web/portal/client/cypress/e2e/Calendar/HolidayDate/HolidayDate.cy.js

```diff
@@ -0,0 +1,159 @@
+import HolidayDateCollection from '../../../fixtures/HolidayDate/getCollection.json';
+import HolidayDateItem from '../../../fixtures/HolidayDate/getItem.json';
+import NewHolidayDate from '../../../fixtures/HolidayDate/post.json';
+import NewHolidayDateRange from '../../../fixtures/HolidayDate/postHolidayDateRange.json';
+import EditHolidayDate from '../../../fixtures/HolidayDate/put.json';
```

Comment:
please follow the convention we are using for naming the fixtures:

```suggestion
import CompaniesItem from '../../../fixtures/Provider/Companies/VirtualPbxs/getItem.json';
import DdiItem from '../../../fixtures/Provider/Ddis/getItem.json';
import newDdi from '../../../fixtures/Provider/Ddis/post.json';
import editDdis from '../../../fixtures/Provider/Ddis/put.json';
```